tests/api2/test_audit_basic.py

```diff
@@ -15,10 +14,60 @@
 
 SMBUSER = 'audit-smb-user'
 PASSWD = ''.join(secrets.choice(string.ascii_letters + string.digits) for i in range(10))
+AUDIT_DATASET_CONFIG = {
+    # keyname : "audit"=audit only setting, "zfs"=zfs dataset setting, "ro"=read-only (not a setting)
+    'retention': 'audit',
+    'quota': 'zfs',
+    'reservation': 'zfs',
+    'quota_fill_warning': 'zfs',
+    'quota_fill_critical': 'zfs',
+    'remote_logging_enabled': 'other',
+    'space': 'ro'
+}
+MiB = 1024**2
+GiB = 1024**3
+
+
+# =====================================================================
+#                     Fixtures and utilities
+# =====================================================================
+class AUDIT_CONFIG():
+    defaults = {
+        'retention': 7,
+        'quota': 0,
+        'reservation': 0,
+        'quota_fill_warning': 75,
+        'quota_fill_critical': 95
+    }
+
+
+# def get_zfs(key, zfs_config):
+def get_zfs(type, key, zfs_config):
```

Comment:
`type` is a python builtin (class).  Maybe better to use a different name instead.